deploy-agent/deployd/agent.py

```diff
@@ -51,6 +53,51 @@ class AgentRunMode(object):
     def is_serverless(mode):
         return AgentRunMode.SERVERLESS == mode
 
+class TimeElapsed():
```

Comment:
this should live independently under common/.

Reply:
moved per req